core/loadpoint.go

```diff
@@ -1173,7 +1173,8 @@ func (lp *Loadpoint) pvMaxCurrent(mode api.ChargeMode, sitePower float64, batter
 
 	if mode == api.ModePV && lp.enabled && targetCurrent < minCurrent {
 		// kick off disable sequence
-		if sitePower >= lp.Disable.Threshold && lp.phaseTimer.IsZero() {
+		sitePowerWithLpIn1pMode := sitePower + minCurrent*float64(1-activePhases)
```

Comment:
Das macht für mich immer noch keinen Sinn.
Wenn der Variablenname korrekt ist, dann können wir als Multiplikator ja direkt 1 (Phase) ansetzen.
Ferner vermisse ich hier die Multiplikation mit Volts. sitePower ist in Watt, minCurrent sind Amps.

Reply:
> Wenn der Variablenname korrekt ist, dann können wir als Multiplikator ja direkt 1 (Phase) ansetzen.

Nein - denn sitePower ist die aktuelle Leistung, der Loadpoint läuft gerade mit minCurrent  und activePhases.
Nach einer Phasenumschaltung läuft er mit minCurrent und 1 Phase.
Die Leistung nach der Umschaltung ändert sich also um Voltage*minCurrent*float64(1-activePhases)

> Ferner vermisse ich hier die Multiplikation mit Volts. sitePower ist in Watt, minCurrent sind Amps.

Gerechtfertigter Einwand - korrigiert. Die Formel hatte noch einen Logikfehler aber jetzt sollte es passen.